Fix accuracy in documentation of Application permissions for azuread_group and azuread_group_member (#1045)
hashicorp/terraform-provider-azuread: docs/resources/group.md

The following API permissions are required in order to use this resource.

> When authenticated with a service principal, this resource requires one of the following application roles: `Group.ReadWrite.All` or `Directory.ReadWrite.All`. Alternatively you can grant the principal the application role `Group.Create`, and make the principal as part of the owners of the group.

Reviewer: ```suggestion
When authenticated with a service principal, this resource requires one of the following application roles: `Group.ReadWrite.All` or `Directory.ReadWrite.All`.

Alternatively, if the authenticated service principal is also an owner of the group being managed, this resource can use the application role: `Group.Create`.
```
